code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -1,4 +1,4 @@
-var/list/hardcoded_gases = list("o2","n2","co2","plasma") //the main four gases, which were at one time hardcoded
+var/list/hardcoded_gases = list("o2","n2","co2","plasma","water vapor","freon","n2o") 
```

Comment:
I dont even know what you're NO'ing about.

This list was used by analyzers and it would omit the other gases from its readout.